src/main/java/org/worldcubeassociation/ui/ExportJSONAction.java

```diff
@@ -45,7 +46,7 @@ public void actionPerformed(ActionEvent aActionEvent) {
         }
 
         try {
-            String jsonFileName = "Results for " + getEnv().getCompetitionId() + ".json";
+            String jsonFileName = "Results (" + WorkbookAssistant.class.getPackage().getImplementationVersion() + ") for " + getEnv().getCompetitionId() + ".json";
```

Comment:
I don't think putting this in the filename is a great idea, as eventually other tools will be producing this JSON (CCM for instance, and maybe cubecomps?).

How about adding a new field to the [WCA Competition JSON](https://github.com/cubing/wca-website/wiki/WCA-Competition-JSON-Format)? There's already a `scrambleProgram`, so we could call it `resultsProgram` or `jsonProgram`.